test/e2e/knative_serving_test.go

```diff
@@ -25,14 +25,16 @@ func TestKnativeServing(t *testing.T) {
 	})
 
 	t.Run("verify correct deployment shape", func(t *testing.T) {
+		servingDeployments := []string{"activator", "autoscaler", "autoscaler-hpa", "controller", "domain-mapping", "domainmapping-webhook", "webhook"}
+
 		// Check the desired scale of deployments in the knative serving namespace
-		for _, deployment := range []string{"activator", "controller", "autoscaler-hpa"} {
+		for _, deployment := range servingDeployments {
 			if err := test.CheckDeploymentScale(caCtx, servingNamespace, deployment, haReplicas); err != nil {
 				t.Fatalf("Failed to verify default HA settings for %q: %v", deployment, err)
 			}
 		}
 		// Check the status of deployments in the knative serving namespace
-		for _, deployment := range []string{"activator", "autoscaler", "autoscaler-hpa", "controller", "webhook"} {
+		for _, deployment := range servingDeployments {
 			if _, err := test.WithDeploymentReady(caCtx, deployment, servingNamespace); err != nil {
 				t.Fatalf("Deployment %s is not ready: %v", deployment, err)
 			}
```

Comment:
Um.. I assumed that your comment in https://github.com/openshift-knative/serverless-operator/pull/1376#discussion_r787495867 suggested the one "loop" as:

```go
	for _, deployment := range servingDeployments {
		// Check the desired scale of deployments in the knative serving namespace
		if err := test.CheckDeploymentScale(caCtx, servingNamespace, deployment, haReplicas); err != nil {
			t.Fatalf("Failed to verify default HA settings for %q: %v", deployment, err)
		}

		// Check the status of deployments in the knative serving namespace
		if _, err := test.WithDeploymentReady(caCtx, deployment, servingNamespace); err != nil {
			t.Fatalf("Deployment %s is not ready: %v", deployment, err)
		}
	}
```

But we don't need to change it? :sweat_smile: If we will make it one loop, we also need to change L#51 below, though.

Reply:
Yes forgot to update it. So tests pass let me do that minor thingy.